src/main/java/codesquad/springcafe/controller/UserController.java

```diff
@@ -0,0 +1,64 @@
+package codesquad.springcafe.controller;
+
+import codesquad.springcafe.dto.User;
+import codesquad.springcafe.service.UserService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import java.util.List;
+import java.util.Optional;
+
+@Controller
+@RequestMapping("/users") // 공유매핑
+public class UserController {
+    private static final Logger logger = LoggerFactory.getLogger(UserController.class);
+    private final UserService userService;
+
+    @Autowired // 생성자가 하나일 때 생략 가능하지만, 우선 남겨둠 ...!
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    // 회원 목록 조회하기
+    @GetMapping("")
+    public String users(Model model){
+        List<User> users = userService.findAllUsers();
+        model.addAttribute("users", users); // users 를 "users" 라는 이름으로 등록 => DB 에서 가져온 데이터를 users 라는 이름으로 뷰 페이지에서 사용 가능
+        return "users/list";
+    }
+
+    // 회원 가입 양식
+    @GetMapping("/signup")
+    public String showSignUpForm(Model model){
+        model.addAttribute("user", new User());
+        return "users/form";
+    }
+
+    // 회원 가입 처리
+    @PostMapping("/signup")
+    public String createUser(@ModelAttribute  User user, RedirectAttributes redirectAttributes){
+        User savedUser = userService.signup(user);
```

Comment:
```suggestion
        User savedUser = userService.signup(userDto);
```
요청에서 만든 객체로 새로운 사용자 객체를 만들어 처리하려면 DTO가 있어야 의미가 있겠죠?
지금은 조금 의미없는 코드가 된 느낌이네요.